src/core/TChem_Driver.cpp

```diff
@@ -118,38 +121,35 @@ void TChem::Driver::freeGasKineticModel() {
   _therm_file = std::string();
 }
 
-void TChem::Driver::createStateVector() {
+void TChem::Driver::createStateVector(ordinal_type nBatch) {
   const ordinal_type len = TChem::Impl::getStateVectorSize(_kmcd_host.nSpec);
-  const ordinal_type nBatch = 1;
   _state = real_type_2d_view_host("state dev", nBatch, len);
```

Comment:
By default, Kokkos initializes views with zeros. Thus, we do not need Kokkos::deep_copy(_state, 0.0).

Reply:
Good to know! I have removed the deep_copy in https://github.com/PCLAeroParams/TChem-atm/pull/55/commits/ddb3bdaeae022332cb89c615c40d9206b318600e